### ISSUE TITLE
Role with no access to topic data - getting error in browser console [0.25.0]

### ISSUE THREAD

**Issue by sverrehu** [tchiotludo/akhq#1784]
I'm trying to create a role that can list topics, but not see the data on the topics. For that I copied the `reader` role, but removed the `TOPIC_DATA` resource:

```yaml
reader-no-data-access:
  - resources: [ "TOPIC", "CONSUMER_GROUP", "CONNECT_CLUSTER", "CONNECTOR", "SCHEMA", "NODE", "ACL", "KSQLDB" ]
    actions: [ "READ" ]
```

However, now the browser console shows an error:

```text
TypeError: Cannot read properties of undefined (reading 'includes')
    at Qf.render (AkhqRoutes.jsx:162:57)
```

I notice the logic in [AkhqRoutes.jsx](https://github.com/tchiotludo/akhq/blob/dev/client/src/utils/AkhqRoutes.jsx#L162):

```javascript
roles && roles.TOPIC && roles.TOPIC_DATA.includes('CREATE')
```

It doesn't look entirely correct. I would expect it to be either `TOPIC` or `TOPIC_DATA` on both the boolean operands, in order not to call `includes` on a non-existing `roles` key.

**Comment by sverrehu** [comment]
Thanks for your kind comment, @AlexisSouquiere! I really think your excellent rewrite of the ACL system should have been marked as a breaking change in the release notes, since I did not realize it was breaking until it broke, and then in the docs I found that it was a breaking change. :-) But that's what you get when there is more than a year between releases, and you eagerly try to upgrade. :-) Thanks anyway, and keep up the good work!
